touchnet/models/llama/processing_llama.py

```
@@ -548,10 +283,71 @@ def batch_text(data, config: DataConfig, tokenizer: BaseTokenizer):
             0, text_len, dtype=torch.int64)
-        buffer["sentence_ids"][cur_batch_idx, cur_text_idx:cur_text_idx + text_len] = cur_sentence_idx
+        buffer["attention_mask"][cur_batch_idx, cur_text_idx:cur_text_idx + text_len] = cur_sentence_idx
         buffer["sentence_lens"][cur_batch_idx, cur_text_idx:cur_text_idx + text_len] = text_len
         buffer["num_sentence"] += 1
         cur_text_idx += text_len
         cur_sentence_idx += 1
     if (not config.dataloader_drop_last_batch) and (cur_text_idx > 0 or cur_batch_idx > 0):
         yield buffer
+
+
+def causal_lm_datapipe(
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)